src/components/NewGame/NewGame.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+import { State, Props } from './NewGame.model';
+import styles from './style.scss';
+
+class NewGame extends React.Component<Props, State> {
+  constructor(props: Props) {
+    super(props);
+    this.state = {
+
+    };
+  }
```

Comment:
If you don't use state in your component - it is a dump component.
You could rewrite it in functional style.